### PR TITLE
wd_helper.au3 ++ _WD_GetBrowserVersion() ++ _WD_GetWebDriverVersion()

### PR DESCRIPTION
**Pull request by mlipok**
These feature have been separated into separate functions to make it possible for the user to check it by themself.

```AutoIt
MsgBox(0, "", _WD_GetBrowserVersion('chrome') & @CRLF & _
		"@error = " & @error & @CRLF & _
		"@extended = " & @extended & @CRLF & _
		"")
MsgBox(0, "", _WD_GetWebDriverVersion(@ScriptDir,'chromedriver.exe') & @CRLF & _
		"@error = " & @error & @CRLF & _
		"@extended = " & @extended & @CRLF & _
		"")

```

### CODE REVIEW

File: wd_helper.au3 [hunk 1]
```diff
@@ -35,7 +35,7 @@
 #cs
 	- Jonathan Bennett and the AutoIt Team
 	- Thorsten Willert, author of FF.au3, which I've used as a model
-	- Michał Lipok (@mLipok) for all his feedback / suggestions
+	- Michał Lipok (@mLipok) for all his feedback / suggestions / contribution
```
Comment (Danp2) [inline wd_helper.au3:38]: You are thanking yourself? LOL

**Comment by Danp2** [comment]
I'll have to take a closer review of your proposed changes, but a few things on my initial observation --

- There are changes to other functions that are unrelated to the described PR. 
- Since the proposed new functions are being "ripped" from _WD_UpdateDriver, listing yourself as the author is a stretch ;-)

File: wd_helper.au3 (hunk 1 above)
Comment (mlipok) [reply wd_helper.au3]: Now I have tweaked a little for a less extensive description.
But I wonder why I do not see here my recent change (from about few minutes ago)

**Review by mlipok — COMMENTED** [review]
ah I see here is my small change (above this post)

**Comment by mlipok** [comment]
> * There are changes to other functions that are unrelated to the described PR.


```autoit
	Local $sData = InetRead($sURL, $iOptions)
	$iErr = $_WD_ERROR_GeneralError

```
this part was modified because InetRead states:
`Failure: | "" (empty string) and sets the @error flag to non-zero.
`
Which mean that you can not be sure which value will be set to @error

**Comment by mlipok** [comment]
> * Since the proposed new functions are being "ripped" from _WD_UpdateDriver, listing yourself as the author is a stretch ;-)

fixed

**Comment by Danp2** [comment]
> this part was modified because helpfile for InetRead says:
> `Failure: | "" (empty string) and sets the @error flag to non-zero. `
> Which mean that you can not be sure which value will be set to @error

That's fine, but IMO it should have been a separate PR because it doesn't have anything to do with the _WD_GetBrowserVersion() and _WD_GetWebDriverVersion(), which is what this PR is concerning.

**Comment by mlipok** [comment]
I hope that now all is fine, because I still learn how github works.

**Comment by mlipok** [comment]
Something went wrong with "Resolve conflict" will check it.

**Pull request merged**